### DISCUSSION TITLE
Cast Specific Cells / Ignore Some Cells while Custom Casting

### DISCUSSION THREAD

**Discussion by panda-avi** [ozlerhakan/poiji#287] [Q&A]
Hi All,

I have commas in some of the cells in my excel which I want to remove. But I only want to remove specific columns only. When I use CustomCasting I do not see an option to apply this on specific columns. 

eg:

```
public class CommaCasting implements Casting {
    @Override
    public Object castValue(Field fieldType, String value, int row, int cell, PoijiOptions options) {
        return value != null ? value.replace(",", "") : null ;
    }
}
```

So suppose if I have an excel as 
```
--------------------------------------------
|  A      |  B              |   C          |
|1,23,00  |  Poiji, Library |  Zero, Cool  |
--------------------------------------------
```

So, I only want to remove comma only from Column A and Column B not from Column C. Can someone help if we have any option available with Poiji. I am using Poiji 3.1.7 but can upgrade to any version supporting this.

Thanks in advance,

**Comment by ozlerhakan** [comment]
Hi @panda-avi , Have you tried out the row and cell parameters explicitly to control cells?

**Reply by panda-avi** [reply]
can you please help me how exactly I can utilise the cell number or row number?

**Reply by ozlerhakan** [reply]
I'll have a look at it

**Reply by ozlerhakan** [reply]
Regarding the example in your question:
```
--------------------------------------------
|  A      |  B              |   C          |
|1,23,00  |  Poiji, Library |  Zero, Cool  |
--------------------------------------------
```
The first row of the column A refers to row:1 column:0
The first row of the column B refers to row:1 column:1
The first row of the column C refers to row:1 column:2

Therefore, you can omit the column C by returning the value itself within your customized Casting class as follows:
```
....
            if (column == 2) {
                return value != null ? value : null;
            }
            return value != null ? value.replace(",", "") : null
....
```
